api/hubble/accounts.mdx

```diff
@@ -0,0 +1,26 @@
+| inflation_destination | string | Deprecated: The account address to receive an inflation payment when they are disbursed on the network.<br /><br />#### Notes:<br />Inflation was discontinued in 2019 by validator vote. |
+| flags | int64 | Denotes the enabling and disabling of certain asset issuer privileges.<br /><br />- Required Field<br /><br />#### Notes:<br />Flags are set on the issuer accounts for an asset. When user accounts trust an asset, the flags applied to the asset originate from this account. Depending on the state table, flags can have various meanings.<br /><br />#### `accounts` and `claimable_balances`:<br /><br /><table> <thead> <tr> <th>Flag</th> <th>Meaning</th> </tr> </thead> <tbody> <tr> <td>0</td> <td>None - Default</td> </tr> <tr> <td>1</td> <td>Auth Required (all trustlines by default are untrusted and require manual trust established)</td> </tr> <tr> <td>2</td> <td>Auth Revocable (allows trustlines to be revoked if account no longer trusts asset)</td> </tr> <tr> <td>4</td> <td>Auth Immutable (all auth flags are read only when set)</td> </tr> <tr> <td>8</td> <td>Auth Clawback Enabled (asset can be clawed back from the user)</td> </tr> </tbody> </table> <h4><code>offers</code>:</h4> <table> <thead> <tr> <th>Flag</th> <th>Meaning</th> </tr> </thead> <tbody> <tr> <td>0</td> <td>Default</td> </tr> <tr> <td>1</td> <td>Passive (offer with this flag will not act on and take a reverse offer of equal price)</td> </tr> </tbody> </table> <h4><code>trust_lines</code>:</h4> <p>Flags are set on the issuer accounts for an asset. When user accounts trust an asset, the flags applied to the asset originate from this account.</p> <table> <thead> <tr> <th>Flag</th> <th>Meaning</th> </tr> </thead> <tbody> <tr> <td>0</td> <td>None, Default</td> </tr> <tr> <td>1</td> <td>Authorized (issuer has authorized account to perform transaction with its credit)</td> </tr> <tr> <td>2</td> <td>Authorized to Maintain Liabilities (issuer has authorized account to maintain and reduce liabilities for its credit)</td> </tr> <tr> <td>4</td> <td>Clawback Enabled (issuer has specified that it may clawback its credit, including claimable balances)</td> </tr> </tbody> </table> |
+| home_domain | string | The domain that hosts this account's stellar.toml file.<br /><br />- Required Field<br /><br />#### Notes:<br />Only applies to asset issuer accounts. The stellar.toml file contains metadata about the asset issuer which helps identify who the issuer is and instills trust in the asset |
```

Comment:
nit: i don't think home domain is actually required. we might want to double check that.